library/std/src/error/tests.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-tidy-end-whitespace
```

Comment:
We should probably not ignore this tidy error. Many many editors will automatically and silently remove trailing whitespace. Many editors also don't visibly show trailing whitespace at all, making it very hard to see what's actually being tested. Git also dislikes trailing whitespace in some cases.

You could use `concat!()`, or end the lines with `\` and start the next one with `\n` to avoid traling whitespace:

```rust
let x = "example  \
\ntrailing whitespace:   \
\n  <- preceding and traling space ->   \
\n...";
```